packages/dataviews/README.md

```diff
@@ -388,9 +388,26 @@ If `selection` and `onChangeSelection` are provided, the `DataViews` component b
 
 A function that determines if a media field or a primary field is clickable. It receives an item as an argument and returns a boolean value indicating whether the item can be clicked.
 
-#### `onClickItem`: `function`
+#### `renderItemLink`: `React.ComponentType`
+
+A render function used to render clickable items.
+
+It can render regular links, but also allows integration with routing libraries (like TanStack Router or React Router).
+
+The component receives the following props:
 
-A callback function that is triggered when a user clicks on a media field or primary field. This function is currently implemented only in the `grid` and `list` views.
+-   `item`: The data item that was clicked
+-   Additional standard HTML anchor props (className, style, etc.)
+
+```jsx
+// Then use it in DataViews
+<DataViews
+	// ...other props
+	renderItemLink={ ( { item, ...props } ) => (
```

Comment:
I'm not sure I'm a big fan of spreading the props. What if we need to add more things that shouldn't be spread?